modules/simple_forms_api/app/services/simple_forms_api/form_upload_notification_email.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  class FormUploadNotificationEmail
+    attr_reader :form_number, :form_name, :first_name, :email, :date_submitted, :confirmation_number,
+                :lighthouse_updated_at, :notification_type
+
+    TEMPLATE_IDS = {
+      confirmation: Settings.vanotify.services.va_gov.template_id.form_upload_confirmation_email,
+      error: Settings.vanotify.services.va_gov.template_id.form_upload_error_email,
+      received: Settings.vanotify.services.va_gov.template_id.form_upload_received_email
+    }.freeze
```

Comment:
A simple refactor can be done to make this more readable:

```suggestion
    template = Settings.vanotify.services.va_gov.template_id
    
    TEMPLATE_IDS = {
      confirmation: template.form_upload_confirmation_email,
      error: template.form_upload_error_email,
      received: template.form_upload_received_email
    }.freeze
```